bob/src/hw_metrics_collector.rs

```diff
@@ -148,6 +151,36 @@ impl DescrCounter {
         //  |  10.000   |      0.006     |
         //  with payload
         //  |  10.000   |      0.018     |
+        if self.lsof_flag {
+            let lsof_str = format!("lsof -a -p {} -d ^mem -d ^cwd -d ^rtd -d ^txt -d ^DEL", process::id());
+            match pipers::Pipe::new(&lsof_str)
+                            .then("wc -l")
+                            .finally() {
+                Ok(proc) => {
+                    match proc.wait_with_output() {
+                        Ok(output) => {
+                            if output.status.success() {
+                                let count = String::from_utf8(output.stdout).unwrap();
+                                if let Ok(count) = count[..count.len() - 1].parse::<u64>() {
```

Comment:
If parsing failed, we should notify user about changed format (and disable lsof)